starlette/datastructures.py

```diff
@@ -13,6 +13,14 @@ class Address(typing.NamedTuple):
     port: int
 
 
+_DefaultType = typing.TypeVar("_DefaultType")
+# Mapping values are covariant, keys are invariant
+_KeyType = typing.TypeVar("_KeyType")  # key type
+_ValueType = typing.TypeVar(
+    "_ValueType", covariant=True
+)  # value type for covariant containers
+
```

Comment:
This PR was using `_KT` and `_VT_co` before my comment here: https://github.com/encode/starlette/pull/1449#pullrequestreview-981163983 😅

I thought it will be more friendly to @tomchristie's eyes, but if more members prefer the convention used in the python docs, then I guess it's fine...

Reply:
I might still suggest dropping some of the redundant comments:

```suggestion
_KeyType = typing.TypeVar("_KeyType")
_ValueType = typing.TypeVar("_ValueType", covariant=True)
```

Reply:
I think the `_VT_co` comes from the fact that in those modules there's also a `_VT`: values in `Mapping` are covariant, values in `Dict` are invariant.

So in this case, _right now_ we could just use `_ValueType` as we are. But once we add types to `MultiDict` (the subclass) I think we'll need an invariant value type. So I'll rename this to `_CovariantValueType`.

And yes, the comments are now redundant.